akka-http-core/src/main/scala/akka/http/scaladsl/model/SslSession.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.scaladsl.model
+
+import akka.http.javadsl.{ model => jm }
+import akka.stream.scaladsl.ScalaSessionAPI
+import javax.net.ssl.SSLSession
+
+class SslSession(val session: SSLSession) extends jm.SslSession with ScalaSessionAPI {
```

Comment:
What's the benefit of wrapping the session at all?

Reply:
To get the `ScalaSessionAPI` scala accessors... I guess we could do without them and expose the SSLSession directly, but then users would have to deal with that java-collection-based null-returning API...

Reply:
Ah missed that. Makes sense to keep that I guess.